data/json/recipes/ammo/40x46mm.json

```diff
@@ -0,0 +1,146 @@
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "launcher", 2 ],
+    "time": "10 m",
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 3 ], [ "manual_shotgun", 3 ] ],
+    "charges": 1,
+    "reversible": true,
+    "using": [ [ "bullet_forming", 2 ], [ "ammo_bullet", 12 ] ],
+    "tools": [ [ [ "press", -1 ] ], [ [ "swage", -1 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "sheet_metal_small", 1 ] ],
+      [ [ "paper", 1 ], [ "wax", 1 ] ],
+      [ [ "gunpowder_shotgun", 30 ] ],
+      [ [ "40x46mm_m118_casing", 1 ] ],
+      [ [ "smpistol_primer", 1 ] ]
+    ]
+  },
+  {
+    "result": "40x46mm_slug_m199",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_SHOT",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "launcher", 2 ],
+    "time": "10 m",
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 3 ], [ "manual_shotgun", 3 ] ],
+    "charges": 1,
+    "reversible": true,
+    "using": [ [ "bullet_forming", 2 ], [ "ammo_bullet", 12 ] ],
+    "tools": [ [ [ "press", -1 ] ], [ [ "swage", -1 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "sheet_metal_small", 1 ] ],
+      [ [ "paper", 1 ], [ "wax", 1 ] ],
+      [ [ "gunpowder_shotgun", 30 ] ],
+      [ [ "40x46mm_m199_casing", 1 ] ],
+      [ [ "lgpistol_primer", 1 ] ]
+    ]
+  },
+  {
+    "result": "40x46mm_flechette_m118",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_SHOT",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "launcher", 2 ],
+    "time": "10 m",
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 3 ], [ "manual_shotgun", 3 ] ],
+    "charges": 1,
+    "reversible": true,
+    "tools": [ [ [ "press", -1 ] ], [ [ "swage", -1 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "sheet_metal_small", 1 ] ],
+      [ [ "paper", 1 ], [ "wax", 1 ] ],
+      [ [ "combatnail", 70 ] ],
+      [ [ "gunpowder_shotgun", 30 ] ],
+      [ [ "40x46mm_m118_casing", 1 ] ],
+      [ [ "smpistol_primer", 1 ] ]
+    ]
+  },
+  {
+    "result": "40x46mm_flechette_m199",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_SHOT",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "launcher", 2 ],
+    "time": "10 m",
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 3 ], [ "manual_shotgun", 3 ] ],
+    "charges": 1,
+    "reversible": true,
+    "tools": [ [ [ "press", -1 ] ], [ [ "swage", -1 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "sheet_metal_small", 1 ] ],
+      [ [ "paper", 1 ], [ "wax", 1 ] ],
+      [ [ "combatnail", 70 ] ],
+      [ [ "gunpowder_shotgun", 30 ] ],
```

Comment:
Looks like you use 
```
      [ [ "sheet_metal_small", 1 ] ],
      [ [ "paper", 1 ], [ "wax", 1 ] ],
      [ [ "gunpowder_shotgun", 30 ] ]
```
quite a few times. Why not make a requirement entry for it, to reduce duplication?

Reply:
It's not really a consistent requirement for 40x46mm reloads, the shotgun type ones just happen to share it.